orchagent/dash/dashorch.h

```diff
@@ -16,28 +18,15 @@
 #include "zmqorch.h"
 #include "zmqserver.h"
 
-struct ApplianceEntry
-{
-    swss::IpAddress sip;
-    uint32_t vm_vni;
-};
-
-struct RoutingTypeEntry
-{
-    std::string action_name;
-    std::string action_type;
-    std::string encap_type;
-    uint32_t vni;
-};
+#include "dash_api/appliance.pb.h"
```

Comment:
Could you pls provide a pointer to PR where you added these headers?

Reply:
These headers were automatically generated by protobuf files and these files are in:  https://github.com/sonic-net/sonic-dash-api/tree/master that was mentioned in the description of this PR.